Dockerfile

```diff
@@ -23,10 +23,30 @@ ARG BASE_IMAGE
 ARG BASE_IMAGE_VERSION
 FROM ${BASE_IMAGE}:${BASE_IMAGE_VERSION}
 
-RUN apk add --no-cache ca-certificates bash expat \
-  && rm -rf /var/cache/apk/*
-
 WORKDIR /
 COPY --from=builder /workspace/manager .
 COPY --from=builder /workspace/daemon ./kruise-daemon
+
+RUN mkdir -p /log /tmp
+RUN chown -R nobody:nobody /log
+RUN chown -R nobody:nobody /tmp
+RUN chown -R nobody:nobody /manager
```

Comment:
plz merge consecutive RUN into a single RUN statement to speed up image building